Resources/Prototypes/Entities/Objects/Devices/Syndicate_Gadgets/war_declarator.yml

```diff
@@ -1,7 +1,7 @@
 - type: entity
   parent: BaseItem
   id: NukeOpsDeclarationOfWar
-  name: the declaration of war
+  name: The declaration of war
```

Comment:
Entity names typically start lowercase.

Reply:
prob shouldnt even have "the" as thats not how we structure names

Reply:
A question on this before I change it back is there meant to be some form of capitalization on the examine or right click menu then or is the convention all meant to be lowercase and just over time capitalizations have been introduced?
I mainly changed this one as it looked really bad as it was pretty much a sentence not starting with a capital.
Eg: this is Capitalized
![image](https://github.com/space-wizards/space-station-14/assets/47093363/9b3ddcb0-0012-4e8f-983b-e9832f6c82c7)
vs this which is not
![image](https://github.com/space-wizards/space-station-14/assets/47093363/d4541063-6d79-479d-9dcb-54f1c5a72d2c)
These are not specific cases it just seems random which ones have capitals and which do not.

Reply:
anyway this is 'declaration of war' now